packages/xo-server/src/api/sr.js

```diff
@@ -826,6 +840,7 @@ reattach.params = {
   nameLabel: { type: 'string' },
   nameDescription: { type: 'string' },
   type: { type: 'string' },
+  deviceConfig: { type: 'object', optional: true },
```

Comment:
Why is it optional?

Reply:
I was trying to not break the API. But actually, since the method was broken, I doubt anyone/anything was using it so we can probably make it non-optional.